R/mod_aboutInfo_utils.R

```diff
@@ -0,0 +1,122 @@
+
+
+
+#' Contributor Card
+#' 
+#' Generate a professional card containing role, a photo, name, site and org
+#'
+#' @param role character, role within the project, usually 'Workstream Lead',
+#'   'Core Contributor', or 'Contributor'
+#' @param pic character, file path to image, 1.3" in width
+#' @param site character, if they have a personal of GitHub site to reference
+#' @param name character, first and last name of the contributor
+#' @param org character, a organization that employs this individual, if
+#'   applicable
+#'
+#' @importFrom shiny em
+#' @importFrom bslib card card_header card_body card_image card_title
+#'
+#' @examples
+#' contrib_card(role = "Workstream Lead",
+#'              pic = 'inst/app/www/images/aaron_clark.png',
+#'              site = 'https://github.com/aclark02-arcus',
+#'              name = "Aaron Clark",
+#'              org = "Arcus Biosciences")
+#'
+#' @keywords internal
+#' @family contrib_cards
+#' 
+contrib_card <- function(role, pic, site, name, org){
+  bslib::card(
+    bslib::card_header(
+      class = "d-flex justify-content-between",
+      role
+    ),
+    bslib::card_body(
+      class = "align-items-center",
+      gap = 0,
+      fillable = FALSE,
+      bslib::card_image(
+        border_radius = "top",
+        file = NULL,
+        src = pic,
+        href = ifelse(is.na(site) | is.null(site), 'https://www.pharmar.org', site),
+        target = "_blank" # doesn't work as hoped
+      ),
```

Comment:
```suggestion
        href = ifelse(is.na(site) | is.null(site), 'https://www.pharmar.org', site)
      ) |>
        tagAppendAttributes(target = "_blank", .cssSelector = "a"),
```
This fixes the target for the URLs.